### PR TITLE
Fixes #35648 - Fixes API documentation typo

### PR DESCRIPTION
**Pull request by elkinaguas**
This pull request fixes a typo in the Foreman API documentation, more specifically in the partition table import functionality.

### CODE REVIEW

File: app/controllers/api/v2/ptables_controller.rb
Comment (stejskalleos) [inline app/controllers/api/v2/ptables_controller.rb:55]: `Import a partition table template` would be IMHO better since we use word template in all other parameters

**Review by stejskalleos — CHANGES_REQUESTED** [review]
One small nit plus commit message should be updated as described in [Foreman contribute](https://theforeman.org/contribute.html#SubmittingPatches)

File: app/controllers/api/v2/ptables_controller.rb
Comment (elkinaguas) [reply app/controllers/api/v2/ptables_controller.rb:55]: I think it would be confusing. The import function imports many features, there included the partition table template. Writing `Import a partition table template` would indicate the function only imports the template. I'm curious to know what others maintainers think.

Comment (ekohl) [reply app/controllers/api/v2/ptables_controller.rb:55]: I'm a big fan of consistency. In listing we have `List all partition tables`:
https://github.com/theforeman/foreman/blob/234d1e80aaafe91622dcf33296ff3d581bca2df1/app/controllers/api/v2/ptables_controller.rb#L12

IMHO it either needs to be `partition table template` everywhere or stick to `partition table`. Given this is making it consistent, I'd consider it a bugfix we can safely cherry pick. We can't do that with a massive rename. So I'd ack this.

Comment (elkinaguas) [reply app/controllers/api/v2/ptables_controller.rb:55]: I'll resolve the conversation if @stejskalleos agrees.

**Comment by elkinaguas** [comment]
Hi @ekohl, is there anything else that need to be done on my side for this PR? I already changed the title to follow the documentation guidelines.

**Comment by ekohl** [comment]
@elkinaguas it is about the git commit itself, not the PR title. You need to amend it and then force push it (and probably rebase it while you're at it). Let me know if you need help with that.

**Comment by ekohl** [comment]
ok to test

**Review by ekohl — APPROVED** [review]
Looks good. I've started CI and once it's green it can be merged.

**Comment by ekohl** [comment]
Cherry picked to 3.5-stable as 42b14720f2984af5cfd06ca2dae6dbcae3fc6e93

**Pull request merged**
